### PR TITLE
Clients(authorization -> evaluate): Adds expandable results table

### PR DESCRIPTION
**Pull request by jenny-s51**
## Motivation

https://marvelapp.com/prototype/5eg9ic7/screen/83188837

## Brief Description

Adds expandable table for results of resource evaluation. 

## Verification Steps

1. Go to Clients
2. Create a new OIDC client and make sure "Authorization" and "Authentication" switches are both enabled.
3. Go to Authorization -> Evaluate
4. Fill out the fields and Click "Evaluate"
5. Verify the data is displayed correctly.

### CODE REVIEW

File: src/clients/authorization/AuthorizationEvaluate.tsx [hunk 1]
```diff
@@ -50,6 +61,12 @@ export type AttributeType = {
   }[];
 };
 
+enum FilterType {
+  allResults = "All results",
```
Comment (edewit) [inline src/clients/authorization/AuthorizationEvaluate.tsx:65]: these need to be translated

File: src/clients/authorization/AuthorizationEvaluate.tsx [hunk 2]
```diff
@@ -332,16 +528,23 @@ export const AuthorizationEvaluate = ({
...
+                control={control}
+                render={() => (
+                  <AttributeInput
+                    selectableValues={resources.map<AttributeType>((item) => ({
+                      name: item.name!,
+                      key: item._id!,
+                    }))}
+                    resources={resources}
+                    isKeySelectable
+                    name="resources"
```
Comment (edewit) [inline src/clients/authorization/AuthorizationEvaluate.tsx:545]: why would you need this controller around this? Can't you just change the name here to 'applyToResource'?

**Review by jonkoops — APPROVED** [review]
Code changes look good, I am running the tests to see if we can merge this.

**Comment by jenny-s51** [comment]
@jonkoops Awesome! I'm currently running the tests locally too.

**Comment by jonkoops** [comment]
All green! :heavy_check_mark: 
```log
       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  authentication_policies.spec.ts          00:15        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  authentication_test.spec.ts              01:05       15       15        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  client_authorization_test.spec.ts        00:22        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  client_scopes_test.spec.ts               00:19        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  clients_saml_test.spec.ts                00:15        5        5        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  clients_test.spec.ts                     00:51       15       15        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  events_test.spec.ts                      00:49       14       14        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  group_test.spec.ts                       00:35        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_oidc_test.spec.t      00:27        5        5        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_saml_test.spec.t      01:02       12       12        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  identity_providers_test.spec.ts          00:42        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  login_test.spec.ts                       00:06        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  masthead_test.spec.ts                    00:15        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  partial_export_test.spec.ts              00:12        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  partial_import_test.spec.ts              00:37        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_roles_test.spec.ts                 00:22        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_client_policies_test      00:45       18       18        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_client_profiles_test      00:43       21       21        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_events_test.spec.ts       01:15       17       16        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_tabs_test.spec.ts         00:24        6        6        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_settings_user_profile_tab.spe      00:12        3        3        -        -        - │
  │    c.ts                                                                                        │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_test.spec.ts                       00:10        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  realm_user_registration.spec.ts          00:08        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  sessions_test.spec.ts                    00:33       11       11        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_kerberos_test.spec.ts           00:44        9        9        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_hardcoded_mapper_test      01:15       12       12        -        -        - │
  │    .spec.ts                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_mapper_test.spec.ts        01:18       12       12        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_fed_ldap_test.spec.ts               00:48        8        8        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  users_test.spec.ts                       00:26       13       12        -        1        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        17:09      251      249        -        2        -  

```

**Pull request merged**